examples/ERC20/index.rsh

```diff
@@ -0,0 +1,146 @@
+// ERC-20 Specification
+// https://eips.ethereum.org/EIPS/eip-20
+//
+// ERC-165
+// https://eips.ethereum.org/EIPS/eip-165
+'reach 0.1';
+
+const ERC165 = mixin({
+  IDs: [ Bytes.fromHex('0x01ffc9a7'), ],
+  View: [{
+    supportsInterface: Fun([Bytes(4)], Bool),
+  }],
+});
+
+const ERC20 = mixin({
+  //IDs: [ Bytes.fromHex('0xTODO'), ],
```

Comment:
!

Reply:
Yeah, I'll compute that today and fix it.